src/uct/ib/rc/base/rc_iface.h

```diff
@@ -201,9 +201,9 @@ struct uct_rc_iface {
     uct_ib_iface_t              super;
 
     struct {
-        ucs_mpool_t             mp;       /* pool for send descriptors */
-        ucs_mpool_t             fc_mp;    /* pool for FC grant pending requests */
-        ucs_mpool_t             flush_mp; /* pool for flush completions */
+        ucs_mpool_t             mp;         /* pool for send descriptors */
+        ucs_mpool_t             pending_mp; /* pool for pending requests */
```

Comment:
minor: maybe mention FC and keepalive explicitly?

Reply:
what if we add new operation for pending request?

Reply:
will add it to the comment). Current comment just repeats the name of the field.
but up to you

Reply:
ok, will add if another comments are found